learning/web/content/modules/benzodiazepines/CON234573_21.mdx

```diff
@@ -46,15 +44,14 @@ import Expander from '../../../src/components/Expander'
 * Substitute the benzodiazepine for another class of anxiolytics
 **Choose your answers as follows:**
```

Comment:
is there a missing line break here?

Reply:
Yes; I haven't got up to Benzodiazepines yet.